kong/runloop/handler.lua

```diff
@@ -91,7 +91,7 @@ do
     local time = ngx.time()
 
     if force or time - last >= LUA_MEM_SAMPLE_RATE then
-      local count = collectgarbage("count")
+      local count = collectgarbage("count") * 1000
```

Comment:
```suggestion
      local count = collectgarbage("count") * 1024
```
Just to be more precise, the original value is split by 1024: `setnumV(L->top, (lua_Number)G(L)->gc.total/1024.0);`

Reply:
Great point!